package/src/components/Message/Message.tsx

```diff
@@ -736,10 +736,14 @@ const MessageWithContext = <
       >
         <View
           style={[
-            { marginTop: 2, paddingHorizontal: screenPadding },
+            { marginTop: 2, paddingHorizontal: screenPadding, ...wrapper },
```

Comment:
Why are we destructuring here ? Just add it at the end of the list as a style extension. Whatever needs to be done from that point on can be by integrators.

Reply:
The purpose was to give `wrapper` styles precedence over `targetedMessageContainer` mainly

Reply:
But I can unify it outside if you think that makes sense

Reply:
You should just need to `[originalStyles, wrapper, targettedMessageStyles]`